en-us/give-up-optifine.md

```diff
@@ -1,36 +1,65 @@
-* Dynamic Lights - [LambDynamicLights](https://www.curseforge.com/minecraft/mc-mods/lambdynamiclights)
-* Connected Textures - [Continuity](https://www.curseforge.com/minecraft/mc-mods/continuity)
-* Animated Textures - [Animatica](https://www.curseforge.com/minecraft/mc-mods/animatica)
-* Custom GUIs - [OptiGUI](https://www.curseforge.com/minecraft/mc-mods/optigui)
-* Custom Items - [CIT Resewn](https://www.curseforge.com/minecraft/mc-mods/cit-resewn)
-* Random Mobs, Emissive Mobs - [Entity Texture Features](https://www.curseforge.com/minecraft/mc-mods/entity-texture-features-fabric)
-* Custom Mobs/Entities - [Custom Entity Models](https://www.curseforge.com/minecraft/mc-mods/custom-entity-models-cem)
-* Custom Colors - [Colormatic](https://www.curseforge.com/minecraft/mc-mods/colormatic)
-* Toggles for animations, particles etc. - [Sodium Extra](https://www.curseforge.com/minecraft/mc-mods/sodium-extra)
-* HD Fonts - support in vanilla since 1.13, some resourcepacks that provide it [are available on my profile](https://www.curseforge.com/members/robotkoer/projects)
-* HD Screenshots - [Fabrishot](https://www.curseforge.com/minecraft/mc-mods/fabrishot)
-* Fix for transparent gaps in items - [Item Model Fix](https://www.curseforge.com/minecraft/mc-mods/item-model-fix)
-* Resource pack-customizable splash screen - [Puzzle](https://www.curseforge.com/minecraft/mc-mods/puzzle)
-
-¹ Snow is opt-in in settings as it changes some textures too, making it weird with some resource packs.
+### Definitons
+
+✔️ = **Supported**
+
+🚧 = **Partly supported**
+
+🔜 = **Soon supported**
+
+❌ = **Not supported**
+
+## Video Options
+
+| Option | Is supported? | Mod providing the feature |
+|-|-|-|
+| Smooth Lighting Level | ❌ | There's [an  issue](https://github.com/FlashyReese/sodium-extra-fabric/issues/125) in the [Sodium Extra mod](https://www.curseforge.com/minecraft/mc-mods/sodium-extra).
+| Dynamic Lights | ✔️ | [Lambda Dynamic Lights](https://www.curseforge.com/minecraft/mc-mods/lambdynamiclights)
+| Shaders | 🚧 | [Iris](https://www.curseforge.com/minecraft/mc-mods/irisshaders) ([Tutorial](./getting-shaders.md)). Some shaders don't work. [PBR textures unsupported](https://discord.com/channels/774352792659820594/774354933436645478/967251726304415784).
+| Details | ✔️ | [Sodium Extra](https://www.curseforge.com/minecraft/mc-mods/sodium-extra) |
+| Animations | ✔️ |  [Sodium Extra](https://www.curseforge.com/minecraft/mc-mods/sodium-extra) |
+| Particles | ✔️ |  [Sodium Extra](https://www.curseforge.com/minecraft/mc-mods/sodium-extra) |
+| 32+ Render Distance | 🔜 | There's [a Pull Request](https://github.com/FlashyReese/sodium-extra-fabric/pull/211) in the [Sodium Extra mod](https://www.curseforge.com/minecraft/mc-mods/sodium-extra). Alternatively, use [Bobby](https://github.com/Fabulously-Optimized/fabulously-optimized/blob/main/INCLUDED-MODS.md#alternative-mods). |
```

Comment:
```suggestion
| 32+ Render Distance | ❌ | Use [Bobby](https://www.curseforge.com/minecraft/mc-mods/bobby) [(why is it not in FO?)](https://github.com/Fabulously-Optimized/fabulously-optimized/issues/46#issuecomment-1067105734). |
```
I think it's better not to mention that PR as the dev said it is very incomplete, so we shouldn't give hopes up that it will come soon.